### PR TITLE
moving files from plot-editor to root folder,

### PR DESCRIPTION
**Pull request by fhnaseer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# JS18_ProjectB_Group4
\ No newline at end of file
+# Skeleton project for Swagger
```
Comment (syxolk) [inline README.md]: You changed the title by accident.

Please add some information about your project, too: How to build, test and run

**Review by syxolk — CHANGES_REQUESTED** [review]
Please improve the README.

**Comment by fhnaseer** [comment]
updated,

**Review by syxolk — APPROVED** [review]

**Comment by syxolk** [comment]
Thanks!

**Pull request merged**